contrib/traceloop.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Traceloop
+
+[Service]
+Type=notify
+NotifyAccess=all
+ExecStartPre=/bin/rm -f /run/traceloop.socket
+ExecStart=/bin/sh -c "/home/kai/kinvolk/traceloop/traceloop serve & while ! curl -fsS --unix-socket /run/traceloop.socket 'http://localhost/list' > /dev/null; do sleep 1; echo Waiting for traceloop to start up; done ; systemd-notify --ready; wait"
```

Comment:
Instead of calling `systemd-notify`, you could do it in Golang:
https://vincent.bernat.ch/en/blog/2017-systemd-golang
https://github.com/coreos/go-systemd/blob/v22.3.2/daemon/sdnotify.go#L56

```
daemon.SdNotify(false, daemon.SdNotifyReady)
```

Reply:
Yes, with ignoring the return code `(false, nil)` it should be fine when used outside of a systemd unit.